src/Avalonia.Controls/Window.cs

```diff
@@ -446,6 +445,24 @@ public PixelPoint Position
             }
         }
 
+        /// <summary>
+        /// Routed event that can be used for global tracking of window destruction.
+        /// </summary>
+        public event EventHandler<RoutedEventArgs>? WindowClosed
```

Comment:
@kekekeks are these events (`WindowOpened`, `WindowClosed`) meant for user consumption? Does it make sense to expose CLR events for them?

Reply:
Reverted changes